index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Document</title>
+</head>
+
+<body>
+  <nav id="navbar">
+    <a href="#profile">profile</a>
+    <a href="#about">about me</a>
+    <a href="#contact">contact</a>

```

Comment:
This was not in the requirements of your project and is only for desktop view. Kindly remove it. The mobile navbar should only contain a hamburger icon.